Project.toml

```
@@ -1,15 +1,13 @@
 name = "Showoff"
 uuid = "992d4aef-0814-514b-bc4d-f2e9a6c4116f"
 author = ["Daniel C. Jones (@dcjones)"]
-version = "1.0.3"
+version = "2.0.0"
```

Why? This isn't a breaking change.

---

I'm under the assumption that for users of Julia < 1.4 it would break? e.g. Ryu isn't available

---

You're increasing the Julia compat bound, users in Julia v1.4 will simply not be able to install this version, there's no breakage. You should only bump the minor version number
```suggestion
version = "1.1.0"
```